gdcdictionary/datamodel_test.py

```diff
@@ -1,2 +0,0 @@
-def test_datamodel():
```

Comment:
Since gdcdatamodel is now removed from the requirements, is there a way to test that the dictionary changes are baseline valid besides deploying it?

Reply:
the tests are moved to [dictioanrytuils](https://github.com/uc-cdis/dictionaryutils/blob/master/run_tests.sh) and called in .travis